.golangci.yml

```diff
@@ -10,7 +10,7 @@ linters-settings:
       - octalLiteral
   govet:
     check-shadowing: true
-  golint:
+  revive:
     min-confidence: 0
```

Comment:
`min-confidence` is not a valid revive setting, https://github.com/mgechev/revive#configuration

Feel free to check the cli but we have some extra rules that don't come enabled by default 

https://github.com/mongodb/mongocli/blob/6ac1f831b2a7d4ae60f4a7d025e4ab3c8da50d79/.golangci.yml#L22-L53

Reply:
updated.